src/utils/sections.ts

```diff
@@ -89,7 +89,7 @@ export const getSuggestionSections = (): Section[] => {
         {
             name: 'NextUp',
             apiMethod: SectionApiMethod.NextUp,
-            itemTypes: 'nextup',
+            itemTypes: 'Episode',
```

Comment:
the previous value was intentionally kept for compatibility for list page

Reply:
reverted